play-frontend-hmrc-play-30/src/main/scala/uk/gov/hmrc/supportfrontend/views/UtilsSupport.scala

```diff
@@ -53,6 +53,12 @@ trait UtilsSupport {
       if (s != null && s.nonEmpty) Some(s) else None
   }
 
+  def nonEmptyStringOrDefault(optValue: Option[String], default: String): String =
+    optValue match {
+      case Some(NonEmptyString(value)) => value
+      case _                           => default
+    }
+
```

Comment:
total stylistic thing, but what about adding some apply methods and I guess maybe an unapply for Option[String] on NonEmptyString for an api like:
```
NonEmptyString("string").getOrElse("default")
NonEmptyString(Some("string")).getOrElse("default")
```

Reply:
hmm , I edited my comment but it looks like it didn't save-  I think I changed my mind as I was reading - the function approach is easier to read I think, rather than apply/unapply